src/current/_includes/common/upgrade-cockroach-version-license-limitations.md

```diff
@@ -0,0 +1,22 @@
+To perform major version upgrades, you must have [a valid license key]({% link {{ page.version.version }}/licensing-faqs.md %}#obtain-a-license).
+
+Patch version upgrades can be performed without a valid license key, with the following limitations:
+
+- The cluster will run without limitations for a 30 day grace period. During that time, alerts are displayed that the cluster needs [a valid license key]({% link {{ page.version.version }}/licensing-faqs.md %}#set-a-license).
+- The cluster is [throttled]({% link {{ page.version.version }}/licensing-faqs.md %}#throttling) at the end of the grace period if no [valid license key is added to the cluster]({% link {{ page.version.version }}/licensing-faqs.md %}#set-a-license) before then.
+
+If you have an **Enterprise Free** or **Enterprise Trial** license, you must enable telemetry using the [`diagnostics.reporting.enabled`]({% link {{ page.version.version }}/cluster-settings.md %}#setting-diagnostics-reporting-enabled) cluster setting as shown below in order to finalize a major version upgrade:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SET CLUSTER SETTING diagnostics.reporting.enabled = true;
+~~~
+
+If a cluster with an **Enterprise Free** or **Enterprise Trial** license is upgraded across patch versions and does not meet telemetry requirements:
```

Comment:
@spilchen same comment as above. Can you confirm that this is the behavior for telemetry when upgrading with a Enterprise Free or Trial license key? I think yes, but making sure.

Reply:
That’s correct, but throttling due to no telemetry won’t happen right away. The cluster will have 7 days to send telemetry before throttling begins. This is because the clock tracking the last telemetry timestamp isn’t persisted, so it starts fresh each time the cluster starts.

Reply:
I think this is covered in the doc by the bullet immediately following that says:

> - The cluster will run without limitations for a 7 day grace period. During that time, alerts are displayed that the cluster needs to send telemetry.